### DISCUSSION TITLE
Remove signing requirement for threshold / multisig keys on account creation

### DISCUSSION THREAD

**Discussion by rhysied** [hiero-ledger/hiero-improvement-proposals#77] [Ideas]
I raised this recently in the Discord however outside of some discussions with Supremax and Bugbytes it didn't gather much traction so thought I'd raise it again here to potentially help with visibility.

Currently a user can take a generated keypair and share the public key with another user who can create an account on their behalf, most importantly, without having to have access to the corresponding secret key. Unfortunately, it is not possible to set up an account using a threshold or multisig key as the current SDKs require you to provide a valid signature for the key (i.e. a 3/5 threshold key requires 3 signatures, 2/2 multisig requires both signatures).

One possible idea was to allow some level of key programmability, such as the ability to remove keys from a keylist rather than having to re-submit the keylist with unwanted keys being removed. Whilst this would help in the case of threshold keys and potentially could have some interesting applications for things like escrow accounts, etc, it doesn't fix the issue of multisig so the best option may be to remove the signing requirement altogether.

**Comment by nkavian** [comment]
Please see this comment / discussion. Multi-party signing is not required on account creation.

https://github.com/hashgraph/hedera-sdk-java/discussions/544#discussioncomment-976069
